### ISSUE TITLE
Bug in Internal Links to Headers

### ISSUE THREAD

**Issue by Justanhy** [typora/typora-issues#4453]
Internal links `[text](#header-title-eg)` are not working for me and bring me to the same point in the document regardless of the header title used in the brackets. For example, 

```
Document [Start](#eg-link)
...
## Eg Link
...
Somewhere in the middle of the Document
...
Document End
```

The link at the start is meant to bring me to `##Eg Link` but brings me to "Somewhere in the middle..." regardless of the header used in the link. This occurs for both links going up and down the document.

Using `<a>` works fine and other web links work as well. Hopefully this is not a duplicate issue because I could not find any like this.

**Working on:**
Windows 10
Typora version 0.10.11

**Comment by abnerlee** [comment]
Could you provide the full sample file? Does this happen in built-in themes?

**Comment by Justanhy** [comment]
> Could you provide the full sample file? Does this happen in built-in themes?

Yes, it happens with the Github Theme as well. Although it seems to only be happening for this particular file, while internal links work for other files.

**Comment by Justanhy** [comment]
[Mathematical Programming Notes (copy).md](https://github.com/typora/typora-issues/files/6597780/Mathematical.Programming.Notes.copy.md)
Here is the sample file, I had to remove all images as they are licensed but the problem still persists.

**Comment by abnerlee** [comment]
should be fixed in 0.11
